jobs/build/ose/scripts/merge-and-build.sh

```diff
@@ -202,11 +212,21 @@ echo
 echo "=========="
 echo "Tito Tagging"
 echo "=========="
-tito tag --accept-auto-changelog
+if [[ "${OSE_VERSION}" == "${OSE_MASTER_BRANCHED}" || "${OSE_VERSION}" == "${OSE_MASTER}" ]]; then
+  declare -a changelog
+  for commit in $( git log "${PREVIOUS_HEAD}..${CURRENT_HEAD}" --pretty=%h --no-merges ); do
+    changelog+=( "--changelog='$( git log -1 "${commit}" --pretty='%s (%ae)' )'" )
+  done
+fi
 export VERSION="v$(grep Version: origin.spec | awk '{print $2}')"
 echo ${VERSION}
-git push
-git push --tags
+set +u
```

Comment:
Actually we still need the steps for the changelog, otherwise those commits brought in by the manual rebase won't be recorded.